src/engine/serverbrowser.cpp

```diff
@@ -309,12 +292,9 @@ void checkresolver()
     int resolving = 0;
     loopv(servers)
     {
-        serverinfo &si = *servers[i];
-        if(si.resolved == serverinfo::RESOLVED) continue;
-        if(si.address.host == ENET_HOST_ANY)
+        if( servers[i]->need_resolve( resolving ) )
         {
-            if(si.resolved == serverinfo::UNRESOLVED) { si.resolved = serverinfo::RESOLVING; resolverquery(si.name); }
-            resolving++;
+            resolverquery(servers[i]->name());
         }
     }
     if(!resolving) return;
```

Comment:
Please replace by some explicit `> 0` check to increase readability.

Reply:
you want "if( some_bool_value > 0 )" ?